modules/common-fs2/src/test/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/config/ConfigFileSpec.scala

```diff
@@ -256,6 +274,75 @@ class ConfigFileSpec extends Specification with CatsIO {
       }
     }
 
+    "parse valid adapter registries" in {
```

Comment:
You could put one remote adapter config in the sample in `config/`. This way you document it for OSS users, and also you can get rid of this new test, that would require to be updated whenever we update the config.